app/packages/operators/src/built-in-operators.ts

```diff
@@ -467,13 +471,24 @@ class SetView extends Operator {
     const inputs = new types.Object();
     inputs.obj("view", { view: new types.HiddenView({}) });
     inputs.str("name", { label: "Name of a saved view" });
+    inputs.str("slug", { label: "Slug of a saved view" });
     return new types.Property(inputs);
   }
   async execute({ hooks, params }: ExecutionContext) {
-    if (params.view) {
-      hooks.setView(params.view);
-    } else if (params.name) {
-      hooks.setViewName(params.name);
+    const { view, name, slug } = params || {};
+    if (view) {
+      hooks.setView(view);
+    } else if (slug) {
+      hooks.setViewName(slug);
+    } else if (name) {
+      const savedViews = hooks.refetchableSavedViews?.[0]?.savedViews;
+      const savedView =
+        Array.isArray(savedViews) &&
+        savedViews.find((view) => name === view.name);
+      if (!savedView) {
+        throw new Error(`Saved view with name "${name}" does not exist.`);
```

Comment:
@imanjra do we have a JS implementation of the [to_slug()](https://github.com/voxel51/fiftyone/blob/fea54b1fb0d23f1a58853204f835f4c4877438e5/fiftyone/core/utils.py#L2373) utility?

Asking because, if so, then you could just do `name = to_slug(name)` here and then `name` could effectively be either a name or slug. The public interface validates that any new saved view name must be unique both as a `name` and a `slug`, so it wouldn't cause any views to be "unreachable".

One consequence of such an approach is that the error message could only read:

```
`Saved view with name or slug "${name}" does not exist`
```

Reply:
Good idea. Yeah, we do have it. I'll use that util here